src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -1607,6 +1607,9 @@ Do you want to install the latest Dynamo update?</value>
   <data name="GroupContextMenuUngroup" xml:space="preserve">
     <value>Ungroup</value>
   </data>
+  <data name="GroupContextMenuGraphLayout" xml:space="preserve">
+    <value>Cleanup Layout</value>
```

Comment:
Do we want this to remain `Cleanup Layout` or make it `Cleanup Group Layout`? Just a suggestion.